modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -302,6 +302,10 @@
       :semantic-version
       (driver.u/semantic-version-gte [4 2])))
 
+(defmethod driver/database-supports? [:mongo :native-specified-collection]
+  [_driver _feature _db]
+  true)
```

Comment:
Supporting a constraint? This would be the first property where the word "supports" sounds strange. At least we should call this features something like `:requires-specified-collection`. Otherwise one might think it's still OK to proceed if one cannot specify the collection.